corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -389,3 +389,12 @@
   default: "no"
   since: "2.51"
   force: true
+
+- name: "Enable asterisk on required question"
+  description: "Adds a red asterisk to denote mandatory questions in a form."
```

Comment:
Should this specify that it's on mobile, since web apps always uses the asterisk for required questions?

Reply:
That's a good point, I thought adding it under the `Android Settings` group was enough, but I understand the confusion with web apps.

Reply:
I missed that it was under Android settings - but I do like the additional text.